### DISCUSSION TITLE
Vue js + Tailwind CSS Github Page: Failed to load resource: the server responded with a status of 404 ()

### DISCUSSION THREAD

**Discussion by dukagjinzejnaj** [community/community#120544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello everyone,

I'm a beginner and I need your help.

I've developed a page with Vue js and Tailwind CSS. In the localhost it works pretty fine. I've created a repository and pushed my project here. The repository is public and the page is published.

**Repository:**
https://github.com/dukagjinzejnaj/product-landing-page

**Page:**
https://dukagjinzejnaj.github.io/product-landing-page/

**Issue:**
When I open my page in the browser it shows a blank page. I've checked the console in my browser and found those errors.
![Screenshot 2024-04-21 144250](https://github.com/community/community/assets/36724473/3bc0ed9f-33bb-48ac-b3b6-ed9a8770d87e)

I've developed similar projects like this, but I didn't use Tailwind CSS. I've created and pushed the github repositories the same way and I didn't have any issues. Since I use Tailwind CSS my page doesn't show.

It would be very helpful if you can help me to find a solution for this issue. You can also check my code if I did something wrong.

Thank you!

**Comment by userkace** [comment]
**Check File Paths:** Double-check the file paths in your HTML code for the Vue components and Tailwind CSS files. Ensure they match the actual location of the files in your project directory. Typos or incorrect paths can cause 404 errors.

**Tailwind CSS Integration:**
- Installation: Double-check that you've installed Tailwind CSS and its dependencies correctly using npm or yarn. Refer to the official Tailwind CSS documentation for installation instructions: https://tailwindcss.com/docs/installation
- Configuration: Ensure your tailwind.config.js file is present and configured properly. This file tells Tailwind which parts of your project to scan for class usage.
- Build Process: If you're using a build process (like Webpack), verify that it's configured to handle Tailwind's directives (like @apply) and generate the necessary CSS files.


**Missing CSS:**
- Import: Check if you're importing the generated Tailwind CSS file (usually named tailwind.css or similar) into your main Vue component (e.g., App.vue). This ensures Tailwind styles are applied to your components.
- Build Output: Make sure the build process correctly outputs the generated Tailwind CSS file to the deployment location. It might be in a different directory than your source code.


**Compare Local vs. Deployed Code:** If possible, compare the generated HTML and CSS files on your local development server with the deployed versions. This might help identify missing files or differences in paths.

From a quick look the deployed website adds 
```html
<script type="module" crossorigin src="/assets/index-DzahVeHw.js"></script>
<link rel="stylesheet" crossorigin href="/assets/index-gz2u4-kD.css">
 ```
underneath the
```html
<title>R400 Laser Presentation Remote</title>
```
on your index.html which is prompting the 404 missing resource as there are no files in your GitHub directory with those names.

**Reply by dukagjinzejnaj** [reply]
Hello @userkace

Thank you for your respond!

I've checked my project and everything is correct.

I've found the main issue. I haven't added the base code in the vite.config.js file.
![Screenshot 2024-04-21 144251](https://github.com/community/community/assets/36724473/a59d3aac-a137-4d78-a2e5-12e4ebf7a84d)

Now it works fine.
